src/graphql/query/shared/hasEverLoggedIn.graphql

```diff
@@ -1,3 +1,13 @@
 query hasEverLoggedIn {
 	hasEverLoggedIn @client
```

Comment:
Generally, I think we should keep this query file clean as it's used in several other context, and has a name indicating exactly what it does. Please use an inline query or create a different file to hold the new data queries.

Reply:
oh, @emuvente already addressed above. Thanks!